tests/pages/formLogin.py

```diff
@@ -0,0 +1,26 @@
+from selenium.webdriver.common.by import By
+
+from .common import CommonOps
```

Comment:
don't make sense to have the same code in tests and src folders, you need to remove it from tests

Reply:
ok, it is done